app/main.py

```diff
@@ -14,4 +14,8 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    counter = 0
+    for char in phrase:
+        if char.lower() == letter.lower():
+            counter += 1
+    return counter
```

Comment:
Try to rewrite your solution to one line with using method `count` of string

Reply:
return phrase.lower().count(letter.lower())

Reply:
[4870564](https://github.com/mate-academy/py-count-occurrences/pull/1473/commits/4870564c0276d23e57bd5a129ec298331c5c45aa)